.github/workflows/build_and_test.yml

```
@@ -47,13 +47,28 @@ on:
         required: false
         type: boolean
         default: true
+      REQUIRED_LABEL:
+        required: false
+        type: string
+        default: 'NONE'
     secrets:
       GOOGLE_CLOUD_GCP:
         required: false
 jobs:
   build_test_deploy:
     runs-on: ${{ inputs.RUNS_ON }}
     steps:
+    - name: does_pr_have_necessary_labels
+      if: ${{inputs.REQUIRED_LABEL != 'NONE' && github.event_name == 'pull_request'}}
```

```suggestion
      if: ${{inputs.REQUIRED_LABEL && github.event_name == 'pull_request'}}
```
Is this working as expected if you do not define any default for `REQUIRED_LABEL`?

---

You are correct, this works. I have adjusted the code!